src/views/Orders.vue

```diff
@@ -246,6 +252,34 @@ export default defineComponent({
         });
       return deleteBatchAlert.present();
     },
+    async skipBatch (batch: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurrence based on the temporal expression.'),
+          buttons: [
+            {
+              text: this.$t("Don't skip"),
+              role: 'cancel',
+            },
+            {
+              text: this.$t('Skip'),
+              handler: async () => {
+                this.store.dispatch('job/skipJob', batch).then((res) => {
+                  if (res.status === 200) {
+                    showToast(translate("This job has been skipped"));
+                  }
+                  else {
+                    showToast(translate("This job schedule cannot be skipped"));
+                    return;
```

Comment:
Why this return is needed?

Reply:
Got it sir! There is no need of return. It should simply show the unsuccessful toast.